### ISSUE TITLE
EACCES error starting 1 Org Network

### ISSUE THREAD

**Issue by m-g-k** [IBM-Blockchain/blockchain-vscode-extension#2156]
On Ubuntu using VS code 1.43.2 and Extension version 1.0.25 I see 2 EACCES errors in VS Code when creating the 1 Org Network - one for the Orderer private key and one for the Org1 private key:

```
Error displaying Fabric Wallets: EACCES: permission denied, open '/home/blockchain/.fabric-vscode/environments/1 Org Local Fabric/wallets/Orderer/admin/0cf429aa5df491c835a5f2f9856820e87ea531a6f7c7aebfc45e346965da7524-priv'

Error displaying Fabric Wallets: EACCES: permission denied, open '/home/blockchain/.fabric-vscode/environments/1 Org Local Fabric/wallets/Org1/admin/4958b19177c5ea63270ffb82d79abe7062206af58deeac3ca53adcfb8dee1197-priv'
```

This was replicated on two different completely clean VMs. No other VS Code actions taken before clicking in Fabric Envrionments to create the 1 Org Network.

When checking the permissions on the files mentioned, they are actually owned by the current logged on user called blockchain in this case:

```
blockchain@ubuntu:~$ ls -al  '/home/blockchain/.fabric-vscode/environments/1 Org Local Fabric/wallets/Org1/admin/4958b19177c5ea63270ffb82d79abe7062206af58deeac3ca53adcfb8dee1197-priv'
-rw-rw-r-- 1 blockchain blockchain 246 Apr  3 19:13 /home/blockchain/.fabric-vscode/environments/1 Org Local Fabric/wallets/Org1/admin/4958b19177c5ea63270ffb82d79abe7062206af58deeac3ca53adcfb8dee1197-priv

blockchain@ubuntu:~$ ls -al '/home/blockchain/.fabric-vscode/environments/1 Org Local Fabric/wallets/Orderer/admin/0cf429aa5df491c835a5f2f9856820e87ea531a6f7c7aebfc45e346965da7524-priv'
-rw-rw-r-- 1 blockchain blockchain 246 Apr  3 19:13 /home/blockchain/.fabric-vscode/environments/1 Org Local Fabric/wallets/Orderer/admin/0cf429aa5df491c835a5f2f9856820e87ea531a6f7c7aebfc45e346965da7524-priv
```
However, the environment seems to be created OK 

![image](https://user-images.githubusercontent.com/11261488/78397873-ed763c00-75e9-11ea-9b59-19762dc5100b.png)

As can be seen it also reported a warning about the extension taking a long time as well but it did connect to the network eventually

I have not seen this with previous releases.

**Comment by Jakeeyturner** [comment]
This problem has been identified and is a result of this [PR](https://github.com/IBM-Blockchain/blockchain-vscode-extension/pull/2074).

Monday afternoon / Tuesday morning I will publish v1.0.26 of the extension to revert this commit, and to fix this problem!

**Comment by Jakeeyturner** [comment]
Fixed in v1.0.26 - closing this issue.
